docs/secret.rst

```diff
@@ -81,7 +81,7 @@ reason, many protocols derive a new key for each session, reset the counter
 to zero with each new key, and never store the derived key or the counter.
 
 You can safely generate random nonces by calling
-:class:``~nacl.utils.random(SecretBox.NONCE_SIZE)``.
+:class:`~nacl.utils.random(SecretBox.NONCE_SIZE)`.
```

Comment:
I'm not sure this is going to work actually, the construction is nonsensical, it's using `:class:`, but the thing it's showing is a function call. I think you actaulyl want to remove the `:class:` and leave the double backticks.